compiler/src/iree/compiler/Codegen/Common/MaterializeTuningSpecsPass.cpp

```diff
@@ -34,6 +34,8 @@
 #define DBGS() (llvm::dbgs() << "[" DEBUG_TYPE "]: ")
 #define LDBG(X) LLVM_DEBUG(DBGS() << X << "\n")
 
+extern llvm::cl::opt<std::string> clTestTarget;
```

Comment:
In general, we should no reference global variables like this. This makes the code hard to maintain and LLVM converged on external storage for flags meant to be external, and having these flags declared in headers.

Here specifically, we should not rely on test flags in this code. All the information we use must come from the gpu target. If the information we need there is not available, we should work on adding it there.